src/containers/FilmFrontpage/FilmSlideshow.tsx

```diff
@@ -20,140 +20,133 @@ interface Props {
+    borderTopRadius: "4px",
+  },
+});
+
+const StyledCarousel = styled(Carousel, {
+  base: {
+    justifyContent: "center",
+    paddingInline: "unset",
+    paddingBlockEnd: "medium",
+    marginBlockStart: "-large",
+    tablet: {
+      marginBlockStart: "-xlarge",
+    },
+    desktop: {
+      marginBlockStart: "-3xlarge",
+    },
+  },
+});
 
 const FilmSlideshow = ({ slideshow }: Props) => {
   const [currentSlide, setCurrentSlide] = useState<GQLFilmSlideshow_MovieFragment>(slideshow[0]!);
+  const [hoverCallback, setHoverCallback] = useState<ReturnType<typeof setTimeout> | undefined>(undefined);
+
+  const onHover = useCallback(
+    (movie: GQLFilmSlideshow_MovieFragment) => {
+      const timeout = setTimeout(() => setCurrentSlide(movie), 500);
+      setHoverCallback(timeout);
+    },
+    [setCurrentSlide],
+  );
 
   return (
     <section>
-      <StyledSafeLink to={currentSlide.path} tabIndex={-1} aria-hidden>
-        <StyledImg
-          src={currentSlide.metaImage?.url ?? ""}
-          sizes="(min-width: 1140px) 1140px, (min-width: 720px) 100vw, 100vw"
-          alt=""
-        />
-        <SlideInfoWrapper>
-          <InfoWrapper>
-            <h3>{currentSlide.title}</h3>
-            <span id="currentMovieDescription">{currentSlide.metaDescription}</span>
-          </InfoWrapper>
-        </SlideInfoWrapper>
-      </StyledSafeLink>
-      <CarouselContainer>
-        <Carousel hideButtons={true}>
+      <SafeLink to={currentSlide.path} tabIndex={-1} aria-hidden>
+        <ImageWrapper>
+          <StyledImage
+            src={currentSlide.metaImage?.url ?? ""}
+            sizes="(min-width: 1140px) 1140px, (min-width: 720px) 100vw, 100vw"
+            alt=""
+          />
+        </ImageWrapper>
+      </SafeLink>
+      <OneColumn wide>
+        <StyledCarousel hideButtons={true}>
           {slideshow.map((movie) => (
-            <FilmCard
+            <StyledSafeLinkCard
+              data-current={movie.id === currentSlide.id}
               key={movie.id}
-              current={movie.id === currentSlide.id}
-              movie={movie}
-              setCurrentSlide={setCurrentSlide}
-            />
+              onMouseDown={(e) => e.preventDefault()}
+              onMouseEnter={() => onHover(movie)}
+              onMouseLeave={() => {
+                if (hoverCallback) {
+                  clearTimeout(hoverCallback);
+                  setHoverCallback(undefined);
+                }
+              }}
+              onFocus={() => setCurrentSlide(movie)}
+              aria-describedby={"currentMovieDescription"}
+              to={movie.path}
+            >
+              <ImageWrapper>
+                <Image src={movie?.metaImage ? movie?.metaImage.url : ""} loading="eager" alt="" />
+              </ImageWrapper>
+              <StyledText textStyle="label.large" fontWeight="bold">
+                {movie.title}
+              </StyledText>
+            </StyledSafeLinkCard>
```

Comment:
Denne var en FilmContentCard før, jeg syntes de var litt for ulike så flyttet den direkte inn i fila her.  Ser nå at den har litt mange styled konstanter over, så kan vurdere å flytte den ut i egen fil. Design komponenten kalles `FilmTeaser` i figma.